includes/rest-api/class-wp-job-manager-filters-status.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Declaration of our Configuration Filter
+ *
+ * @package WPJM/REST
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Class WP_Job_Manager_Models_Settings
+ */
+class WP_Job_Manager_Filters_Status extends WPJM_REST_Model_Declaration {
+
+	/**
+	 * Declare our fields
+	 *
+	 * @param  WPJM_REST_Model_Field_Declaration_Collection_Builder $def Def.
+	 * @return array
+	 * @throws WPJM_REST_Exception Exc.
+	 */
+	function declare_fields( $def ) {
+		return array(
+		 $def->field( 'keys', 'The status keys to return' )
+			 ->typed( $def->type( 'array:string' ) )
+			 ->before_set( 'explode_keys' )
+			 ->with_default( array() ),
+		);
+	}
+
+	/**
+	 * Explode keys
+	 *
+	 * @param  MT_Interfaces_Model $model Model.
+	 * @param  mixed               $keys  The keys.
+	 * @return array
+	 */
+	function explode_keys( $model, $keys ) {
```

Comment:
This whole Model class is for explicitly modelling the query params accepted by the get / of the controller, which in our case accepts a `keys` query param with comma-separated status keys to return.

 I am leaning towards leaving it here, but definitely possible to put it in the abstract declaration class (possibly with a more generic name `array_from_comma_separated_string`)